nautobot/ipam/tests/test_filters.py

```diff
@@ -1249,3 +1254,278 @@ def test_virtual_machine(self):
+
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="fd78:da4f:e596:c217::/64").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="fd78:da4f:e596:c217::/120").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contained_or_equal="fd78:da4f:e596:c218::/64").count(), 0)
+
+    def test_net_contains(self):
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.0.0.0/8").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.0.0/16").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.0/24").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.192/28").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.192/30").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="192.168.3.192/32").count(), 3)
+
+        self.assertEqual(Prefix.objects.filter(network__net_contains="fd78:da4f:e596:c217::/64").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="fd78:da4f:e596:c217::/120").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="fd78:da4f:e596:c217::/122").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contains="fd78:da4f:e596:c218::/64").count(), 0)
+
+    def test_net_contains_or_equals(self):
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="192.0.0.0/8").count(), 0)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="192.168.0.0/16").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="192.168.3.0/24").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="192.168.3.192/28").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="192.168.3.192/30").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="192.168.3.192/32").count(), 3)
+
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="fd78:da4f:e596:c217::/64").count(), 1)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="fd78:da4f:e596:c217::/120").count(), 2)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="fd78:da4f:e596:c217::/122").count(), 3)
+        self.assertEqual(Prefix.objects.filter(network__net_contains_or_equals="fd78:da4f:e596:c218::/64").count(), 0)
+
+    def test_get_by_prefix(self):
+        prefix = Prefix.objects.filter(network__net_equals="192.168.0.0/16")[0]
+        self.assertEqual(Prefix.objects.get(prefix="192.168.0.0/16"), prefix)
+
+    def test_get_by_prefix_fails(self):
+        _ = Prefix.objects.filter(network__net_equals="192.168.0.0/16")[0]
```

Comment:
Seems unnecessary?

Reply:
I did not come up with this pattern, just following what was in core. No issues removing, just pointing out https://github.com/nautobot/nautobot/blob/a97be77462e6c8b30877e25f31a7263dc433f468/nautobot/ipam/tests/test_querysets.py#L66

Reply:
I'll presume following the core pattern is valid.